src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat.pop());
+  const correctDate = [];
+  let tempString = '';
+
+  for (let i = 0; i < 4; i++) {
+    for (let j = 0; j < 3; j++) {
+      if (fromFormat[j].slice(0, 2) === toFormat[i].slice(0, 2)) {
```

Comment:
can you think about simplier logic? I think it's possible not to use loop inside loop

- you can read about algorithm complexity and why it's better to avoid having loop inside loop 

i think i's a good task to use object properties as well 😉